packages/forms/src/__tests__/addressLookup.spec.tsx

```diff
@@ -122,58 +122,64 @@ describe('Address lookup', () => {
 				render: <AddressLookup {...defaultProps} />,
 			});
 
-			await searchForAPostcode(FakeAddressLookupProvider.tprAddress.postcode);
+			await act(async () => {
+				await searchForAPostcode(FakeAddressLookupProvider.tprAddress.postcode);
 
-			const displayedPostcode = await findByText(
-				FakeAddressLookupProvider.tprAddress.postcode,
-			);
-			expect(displayedPostcode).toBeDefined();
+				const displayedPostcode = await findByText(
+					FakeAddressLookupProvider.tprAddress.postcode,
+				);
+				expect(displayedPostcode).toBeDefined();
 
-			const selectAddressInput = await findByTestId('select-address-list');
-			selectAddressInput.click();
+				const selectAddressInput = await findByTestId('select-address-list');
+				selectAddressInput.click();
 
-			const addressOptions = await findAllByRole('option');
+				const addressOptions = await findAllByRole('option');
 
-			expect(addressOptions[0].textContent).toMatch(
-				FakeAddressLookupProvider.tprAddress.addressLine1,
-			);
+				expect(addressOptions[0].textContent).toMatch(
```

Comment:
Assertions should be outside `act()` - see https://reactjs.org/docs/testing-recipes.html#act

Reply:
done